src/cdk/a11y/a11y.md

```diff
@@ -130,13 +130,6 @@ to the element when focused. It will add `.cdk-focused` if the element is focuse
 add `.cdk-${origin}-focused` (with `${origin}` being `mouse`, `keyboard`, `touch`, or `program`) to
 indicate how the element was focused.
 
-Note: currently the `FocusMonitor` emits on the observable _outside_ of the Angular zone. Therefore
```

Comment:
Was this documentation just out of date? I see that you've removed it, but I don't see a corresponding code change

Reply:
It seems to have been out of date.